salt/fileserver/hgfs.py

```diff
@@ -310,7 +301,7 @@ def init():
             _failhard()
 
         try:
-            refs = repo.config(names="paths")
```

Comment:
Do we need to drop the keyword argument for a positional argument here?

Reply:
I looked in the library `names` is the first arg but I cant add it back